docs/discover/context.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[discover-document-context]]
+== View a document in context
+
+Once you've narrowed your search to a specific event,
+you might want to inspect the documents that occurred
+immediately before and after the event. With the Context view,
+you can do just that for index patterns that contain time-based events.
+
+To open the Context view, click the expand icon (>) in the document table, and then click
+*View surrounding documents.*
+
+The documents are sorted  by the time field specified in the index pattern 
+and displayed using the same set of columns as the *Discover* view from which 
+the context was opened. The anchor document is highlighted in blue.
+
+
+[role="screenshot"]
+image::images/Discover-ContextView.png[Image showing context view feature, with anchor documents highlighted in blue]
+
+[float]
+[[filter-context]]
+=== Filter the context
+
+The filters you applied in *Discover* are carried over to the Context view. 
+Pinned filters remain active, while normal filters are copied in a disabled state. 
+You can re-enable these filters to refine your context view.
+
+If the Context view contains a large number of documents not related to the event under
+investigation, you can use filters to restrict the documents to display.
+
+[float]
+[[change-context-size]]
+=== Change the number of surrounding documents
+
+By default, the five newest and oldest
+documents are listed. To increase the number of documents that surround the anchor document,
+click *Load*.  Five documents are added with each click.
```

Comment:
Shoud we include a screenshot of where the Load button is?

Reply:
This was added